app/models/product.rb

```diff
@@ -12,8 +12,8 @@ class Product < ApplicationRecord
 
 
   enum gender: {
-    "female": 0,
-    "male": 1
+    "Female": 0,
+    "Male": 1
```

Comment:
Thanks for cleaning this up! I totally forgot about this doing the meow/products/view on my side and called `.capitalize` on it haha